client/starwhale/core/dataset/type.py

```diff
@@ -335,6 +330,7 @@ def __init__(
         as_mask: bool = False,
         mask_uri: str = "",
         dtype: t.Type = numpy.uint8,
+        **kwargs: t.Any,
```

Comment:
why add `kwargs` in these functions, but the inner code does not use the kwargs?

Reply:
> why add `kwargs` in these functions, but the inner code does not use the kwargs?

Make sure all subclass of `BaseArtifact` could use the feature of  `BaseArtifact` especially `link` when constructing objects.

Reply:
**kwargs is replaced with link